contracts/crowdsale/ZxcCrowdsale.sol

```diff
@@ -97,24 +107,28 @@ contract ZxcCrowdsale is
    * @dev Contract constructor.
    * @param _walletAddress Address of the wallet which collects funds.
    * @param _tokenAddress Address of the ZXC token contract.
+   * @param _xcertKycAddress Address of the Xcert KYC token contract.
    * @param _startTimePresale Start time of presale stage.
    * @param _startTimeSaleWithBonus Start time of public sale stage with bonus.
    * @param _startTimeSaleNoBonus Start time of public sale stage with no bonus.
    * @param _endTime Time when sale ends.
    * @param _rate ZXC/ETH exchange rate.
-   * @param _crowdSaleZxcCap Number of ZXC tokens offered during the sale.
+   * @param _presaleZxcCap Maximum number of ZXC offered for the presale.
+   * @param _crowdSaleZxcSupply Supply of ZXC tokens offered for the sale. Includes _presaleZxcCap.
    * @param _bonusPresale Bonus token percentage for presale.
    * @param _bonusSale Bonus token percentage for public sale stage with bonus.
    * @param _minimumWeiDeposit Minimum required deposit in wei.
    */
   constructor(address _walletAddress,
```

Comment:
First argument should be in a row of its own.

Reply:
e82ca41